src/main/java/org/folio/notes/domain/repository/NoteTypesRepository.java

```diff
@@ -12,7 +12,7 @@
 
 public interface NoteTypesRepository extends JpaCqlRepository<NoteTypeEntity, UUID> {
 
-  @Query("SELECT n.type.id as typeId, COUNT(n.id) as count FROM NoteEntity AS n WHERE n.type.id IN (:noteTypeIds) GROUP BY n.type.id")
+  @Query("SELECT t.id as typeId, EXISTS (SELECT n.type.id FROM NoteEntity AS n WHERE n.type.id = t.id) as isAssigned FROM NoteTypeEntity as t WHERE t.id IN (:noteTypeIds)")
```

Comment:
```suggestion
  @Query("SELECT t.id as typeId, EXISTS (SELECT n.type.id FROM NoteEntity AS n WHERE n.type.id = t.id LIMIT 1) as isAssigned FROM NoteTypeEntity as t WHERE t.id IN (:noteTypeIds)")
```
It's enough to have at least one record  in the subquery